website/content/docs/consul-vs-other/dns-tools-compare.mdx

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+page_title: Consul compared to other DNS tools 
+description: >-
+  Service discovery is one of Consul's foundational capabilities. Consul is platform agnostic, which allows it to discover services across multiple runtimes and cloud providers including VMs, bare-metal, Kubernetes, EKS, AKS, ECS, Nomad, and Lambda.
+---
+
+
+# Consul Compared to Other DNS Tools
+
+**Examples**: NS1, AWS Route53, AzureDNS, Cloudflare DNS
+
+Consul was originally designed as a centralized service registry for any cloud environment that dynamically tracks services as they are added, changed, or removed within a compute infrastructure. Consul maintains a catalog of these registered services and keeps track of service attributes, such as IP addresses or service name. For more information, refer to [What is Service Discovery?(/docs/intro/usecases/what-is-service-discovery).
+
+As a result, Consul can also provide basic DNS functionality, including [lookups, alternate domains, and access controls](/docs/discovery/dns). And because Consul is platform agnostic, it can provide these capabilities in both cloud and on-premises data centers. Consul does not support some advanced DNS capabilities, such as filters or advanced routing logic. But you can integrate Consul with some existing DNS solutions, such as [NS1](https://help.ns1.com/hc/en-us/articles/360039417093-NS1-Consul-Integration-Overview) and [DNSimple](https://blog.dnsimple.com/2022/05/consul-integration/), to support these advanced capabilities. 
```

Comment:
```suggestion
As a result, Consul can also provide basic DNS functionality, including [lookups, alternate domains, and access controls](/docs/discovery/dns). Since Consul is platform agnostic, you can retrieve service information across both cloud and on-premises data centers. Consul does not natively support some advanced DNS capabilities, such as filters or advanced routing logic. However, you can integrate Consul with existing DNS solutions, such as [NS1](https://help.ns1.com/hc/en-us/articles/360039417093-NS1-Consul-Integration-Overview) and [DNSimple](https://blog.dnsimple.com/2022/05/consul-integration/), to support these advanced capabilities. 
```